scripts/datetime_phrases.py

```
@@ -0,0 +1,66 @@
+import random
```

Ruff (0.7.0)

`random` imported but unused

Remove unused import: `random`

(F401)